apps/web/src/services/cost-explorer/budget/model.ts

```diff
@@ -72,3 +72,18 @@ export interface BudgetUsageModel {
     data_source_id: string;
     updated_at: string;
 }
+
+export interface BudgetUsageAnalyzeModel {
+    budget_id: string;
+    budget_usage: number;
+    total_budget: number;
+    total_spent: number;
+    name?: string;
+    project_id?: string;
+    project_group_id?: string;
+    data_source_id?: string;
+    provider_filter?: {
+        state?: string;
+        providers?: string[];
+    };
+}
```

Comment:
suggestion: I think this is not a fixed resource model. this is just a response spec for `BudgetList` component.